src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1998,6 +2103,40 @@ namespace FanCoilUnits {
 						Par( 4 ) = QZnReq;
 						SolveRegulaFalsi( 0.001, MaxIterCycl, SolFlag, PLR, CalcFanCoilLoadResidual, 0.0, 1.0, Par );
 					}
+				} else {
+					// demand greater than capacity
+					if ( FanCoil( FanCoilNum ).HCoilType_Num == HCoil_Water ) {
+						HWFlow = MaxWaterFlow;
+					} else{
+						Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOut, 1.0 );
+					}
```

Comment:
What does this else do? If demand > capacity, yes to full flow. But don't you need to call Calc4PipeFanCoil regardless?